Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByMimeType.java

```diff
@@ -212,6 +207,37 @@ public static boolean isEmptyMimeTypeNode(Node node) {
 
     }
 
+    private void refreshMimeTypes() {
+        /**
+         * Checking for a current case is a stop gap measure until a different
+         * way of handling the closing of cases is worked out. Currently, remote
+         * events may be received for a case that is already closed.
+         */
+        try {
+            Case.getCurrentCaseThrows();
+            typesRoot.updateShowCounts();
+            populateHashMap();
+        } catch (NoCurrentCaseException notUsed) {
+            /**
+             * Case is closed, do nothing.
+             */
+        }
+    }
+
+    private class FileTypesByMimeTypeRefresher implements RefreshThrottler.Refresher {
+
+        @Override
+        public void refresh() {
+            refreshMimeTypes();
+        }
+
+        @Override
+        public boolean isRefreshRequired(PropertyChangeEvent evt) {
```

Comment:
Did we come to the conclusion that there was no point in checking to see if the file actually has a mime type before refreshing?

Reply:
Yes